introduction.tex

```diff
@@ -8,24 +8,24 @@ \section{Introduction}\label{section:introduction}
 
 \noindent
 \textbf{Workflow.}
-A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things his receiving address on the target blockchain. The user then sends an amount of the source cryptocurrency to that address, essentially burning this amount in the source blockchain. He then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on his receiving address. The equivalent amount can be determined based to the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
+A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things her receiving address on the target blockchain as metadata. The user then sends an amount of the source cryptocurrency to that address, burning this amount in the source blockchain. She then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on her receiving address. The equivalent amount can be determined based on the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. Upon burning her coins the user can generate a proof-of-burn which is a string which can be verified by any interested party given the metadata. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
```

Comment:
Shorten:
* The user -> She
* this amount -> it
* She then proceeds to submit -> She submits
* where the burn is verified and the user -> where it is verified and she
* can be determined -> is determined
* the user -> she
* can generate -> generates
* a proof-of-burn which is a string which can be verified by any interested party given the metadata -> a proof-of-burn, a string which can be verified by any interested party given the metadata
* The verification of the proof of the burn can happen -> Proof of burn verification happens

Reply:
these fixes are applied to #32 as the sections are ultimately moved